assignment1/isPermutation.py

```diff
@@ -0,0 +1,28 @@
+def isPermutation(str1, str2):
+    if str1 is None or str2 is None:
+        return False
+    str1 = str1.lower()
+    str2 = str2.lower()
+    if str1 == str2:
+        return True
+    allSymbols = dict()
```

Comment:
The meaning of `allSymbols` is not clear. It is a dictionary, but what are the keys and what are the values?
You can either find self-describing name or add a comment above this line to describe what it is.